### DISCUSSION TITLE
Registration of landline numbers no longer working?

### DISCUSSION THREAD

**Discussion by pbiering** [AsamK/signal-cli#1429] [Q&A] [answered]
Has anyone successfully registered a landline number in the last weeks?

I have no luck anymore here, following the documented steps

1. register landline number without --voice -> `Failed to register: StatusCode: 502 (ExternalServiceFailureException)`
2. register landline numbe now with --voice -> `Failed to register: StatusCode: 429 (RegistrationRetryException)`

Tried also the hint found in Internet using a "free" mobile phone, install Signal app and specify landline number, here also similar messages appear, first something like 502, then 429 with a delay when I can try next...then the loop continues :-(

**Comment by AsamK** [comment]
I had success just a few days ago. I also got a 502, retried the next day and it worked.

**Reply by AsamK** [reply]
Related: #1373

**Comment by Pilot001** [comment, marked as answer]
Yes, I did just 3 weeks ago...
Please do _exactly_ as it is explained. And you really have to wait the necessary time between the two tries.

**Comment by pbiering** [comment]
Sorry for late reply, I replayed it again and it worked now as expected. No clue why I had the problems but I read somewhere else that one should avoid configuring on an already registered system a 2nd account - potentially this also appeared here during the tests. In this case: use a temporary "fresh" system, register and then transfer the account details to the multi-account system and merge `accounts.json` files.

BTW: I need a multi-account system because of monitoring and letterbox sensor server are running on same system but alerts and notifications should have different senders for being able to configure different sounds on phone side and also have here different senders visible.
